src/Android/Avalonia.Android/Platform/SkiaPlatform/TopLevelImpl.cs

```diff
@@ -50,6 +46,7 @@ class TopLevelImpl : IAndroidView, ITopLevelImpl, EglGlPlatformSurface.IEglWindo
         private readonly ClipboardImpl _clipboard;
         private ViewImpl _view;
         private WindowTransparencyLevel _transparencyLevel;
+        private readonly object[] _surfaces;
```

Comment:
I am pretty sure rendering backend calls this property only once anyway.
But if we pre-set it, then field can be avoided. You can set property directly.